spec/models/withdrawal_spec.rb

```diff
@@ -115,8 +125,8 @@ def create_withdrawal_without_callback
       it 'should raise validation error' do
         expect do
           create(:withdrawal,
-            user: user_with_invoice,
-            btc_address: '1AGNa15ZQ0AZUgF1qJ2i7Z2DPU2J6hW62i')
+                 user: user,
```

Comment:
Use one level of indentation for parameters following the first line of a multi-line method call.